extensions/shared/external-media/sources/icons.js

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
Part of me wonders if we shouldn't have a new single file in `shared` that would list numerous icons like the ones below as well as the Jetpack icon from `extensions/shared/jetpack-logo.js`.